core/templates/domain/skill/skill-summary.model.ts

```diff
@@ -21,8 +21,8 @@ export interface SkillSummaryBackendDict {
   'description': string;
   'language_code': string;
   'version': number;
-  'misconception_count': number;
-  'worked_examples_count': number;
+  'misconception_count': number | null;
+  'worked_examples_count': number | null;
```

Comment:
Why can these be `null`?

Reply:
I assigned it to `null` because a a `skillBackendSummaryDict` object was defined with typed `BackendSummaryDict` and the `misconception_count` property was initialized to `null`. This created an error type of null is not assignable to type number
That's why I went to `core/templates/domain/skill/skill-summary.model.ts` and changed the type to `number | null`

Reply:
Okay, but what does it conceptually mean when these are `null`?  Maybe look into the backend code and see if they can be `null` (or `None` as we use Python in the backend).

Reply:
Hello @vojtechjelinek  I have looked at the model for SkillSummary and seen the clear definitions of these properties
`misconception_count` 
`worked_example_counts`
in the following file: `core/storage/skill/gae_models.py`

![image](https://user-images.githubusercontent.com/37558983/160793715-46878f90-6214-4057-b233-90a8b12b03ea.png)

Reply:
One of my suggestions was to avoid these nulls by assigning the `misconception_count =  0` and the `worked_examples_count = 0` in the `core/templates/components/skill-selector/select-skill-modal.component.spec.ts` instead of `null, null`

![image](https://user-images.githubusercontent.com/37558983/160795915-4c9852f3-8da5-4e98-92ed-bbf829066f6a.png)

Reply:
Okay, so it seems that in backend we set some value to them, maybe you can try to find the default values in the backend and then use them in the frontend (I'm not sure if it is 0).